app/views/userProfile.scala.html

```diff
@@ -14,22 +14,26 @@
             <div class="row" id="profile-container">
                 <div id="username">@user.get.username</div>
                 <div id="org-container">
-                    @if(!UserOrgTable.getAllOrgs(user.get.userId).isEmpty) {
-                        <div id="org-name">@OrganizationTable.getOrganizationName(UserOrgTable.getAllOrgs(user.get.userId).head)</div>
-                    } else {
-                        <div id="org-name">@Messages("dashboard.no.org")</div>
-                    }
-                    <div id="edit-org-dropdown" class="dropdown" data-toggle="popover" data-trigger="hover" data-placement="right" data-content="@Messages("dashboard.edit.info")">
-                        <a id="edit-org-button" role="button" data-toggle="dropdown" href="#">@Messages("dashboard.edit.org") <b class="caret"></b></a>
+                    <div id="edit-org-dropdown" class="dropdown" data-toggle="popover" data-trigger="hover" data-placement="right">
+                        @if(!UserOrgTable.getAllOrgs(user.get.userId).isEmpty) {
+                            <a id="edit-org-button" role="button" data-toggle="dropdown" href="#">
```

Comment:
It looks like the only thing that's different between the two conditions is the text, so I think that we could simplify by writing it like this?
```
<a id="edit-org-button" role="button" data-toggle="dropdown" href="#">
    @if(!UserOrgTable.getAllOrgs(user.get.userId).isEmpty) {
        @OrganizationTable.getOrganizationName(UserOrgTable.getAllOrgs(user.get.userId).head) <b class="caret"></b>
    } else {
        @Messages("dashboard.join.team") <b class="caret"></b>
    }
</a>
```